cluster/juju/prereqs/ubuntu-juju.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Copyright 2014 Canonical LTD. All rights reserved.
```

Comment:
I don't think we can take code that has anything other than the Kubernetes boilerplate at the top.  See comment on conversation tab about github hooks.